<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordChangeForm</class>
 <widget class="QWidget" name="PasswordChangeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="GroupBoxEx" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>70</number>
      </property>
      <item row="0" column="0">
       <widget class="LabelEx" name="label">
        <property name="text">
         <string>旧密码：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="LineEditEx" name="lineEditPasswdOld">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="LabelEx" name="label_2">
        <property name="text">
         <string>新密码：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="LineEditEx" name="lineEditPasswdNew">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="LabelEx" name="label_3">
        <property name="text">
         <string>确认密码：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="LineEditEx" name="lineEditPasswdMakeSure">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="PushButtonEx" name="pushButtonCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="PushButtonEx" name="pushButtonOk">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupBoxEx</class>
   <extends>QGroupBox</extends>
   <header>ctrls/GroupBoxEx.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditEx</class>
   <extends>QLineEdit</extends>
   <header>ctrls/LineEditEx.h</header>
  </customwidget>
  <customwidget>
   <class>LabelEx</class>
   <extends>QLabel</extends>
   <header>ctrls/LabelEx.h</header>
  </customwidget>
  <customwidget>
   <class>PushButtonEx</class>
   <extends>QPushButton</extends>
   <header>ctrls/PushButtonEx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
